Makefile

```diff
@@ -234,21 +234,23 @@ build-package-deb-arm64:
 	@cd $(CURRENT_DIR)/axolotl-$(AXOLOTL_VERSION) && debuild -i -us -uc -b
 
 install-deb-arm64: uninstall-deb-arm64
-# Use for testing purposes only after prebuild-package-arm64
+# Use for testing purposes only
 	@echo "Installing Axolotl"
 # Copy libzkgroup
-	@sudo $(WGET) https://github.com/nanu-c/zkgroup/raw/main/lib/libzkgroup_linux_aarch64.so -P /usr/lib
-	@sudo mkdir -p $(DESTDIR)$(SHARE_PREFIX)/axolotl
-	@sudo cp -r $(CURRENT_DIR)/axolotl-$(AXOLOTL_VERSION)/axolotl/* $(DESTDIR)$(SHARE_PREFIX)/axolotl
-	@sudo mv $(DESTDIR)$(SHARE_PREFIX)/axolotl/axolotl $(DESTDIR)$(INSTALL_PREFIX)/
-	@sudo cp $(CURRENT_DIR)/deb/axolotl.desktop $(DESTDIR)$(SHARE_PREFIX)/applications
-	@sudo cp $(CURRENT_DIR)/deb/axolotl.png $(DESTDIR)$(SHARE_PREFIX)/icons/hicolor/128x128/apps
+	@sudo $(WGET) https://github.com/nanu-c/zkgroup/raw/main/lib/libzkgroup_linux_aarch64.so -P $(DESTDIR)$(LIBRARY_PREFIX)/
+#	Copy binary and helpers
+  @sudo mkdir -p $(DESTDIR)$(SHARE_PREFIX)/axolotl
+	@sudo cp -r $(CURRENT_DIR)/build/linux-arm64/* $(DESTDIR)$(SHARE_PREFIX)/axolotl/
+  @sudo mv $(DESTDIR)$(SHARE_PREFIX)/axolotl/axolotl $(DESTDIR)$(INSTALL_PREFIX)/
+	@sudo cp $(CURRENT_DIR)/deb/axolotl.desktop $(DESTDIR)$(SHARE_PREFIX)/applications/
+	@sudo cp $(CURRENT_DIR)/deb/axolotl.png $(DESTDIR)$(SHARE_PREFIX)/icons/hicolor/128x128/apps/
 	@sudo cp $(CURRENT_DIR)/deb/axolotl.sh /etc/profile.d
 	@bash -c "source /etc/profile.d/axolotl.sh"
 	@echo "Installation complete"
 
 uninstall-deb-arm64:
-	@sudo rm -rf $(DESTDIR)$(SHARE_PREFIX)/axolotl
+	@pkill axolotl
+  @sudo rm -rf $(DESTDIR)$(SHARE_PREFIX)/axolotl/
```

Comment:
the whitespace is wrong here

Reply:
True! I wonder, how that happenend... must be magic.

Reply:
Should be good now.